packages/java-parser/src/productions/expressions.js

```diff
@@ -309,12 +309,7 @@ function defineRules($, t) {
       $.SUBRULE($.annotation);
     });
 
-    $.OPTION({
-      NAME: "$methodTypeArguments",
-      DEF: () => {
-        $.SUBRULE2($.typeArguments);
-      }
-    });
+    $.OPTION(() => $.SUBRULE2($.typeArguments));
```

Comment:
this change is not related to the upgrade correct?

Reply:
Support for nested In line rules has been deprecated in v7.0.0, if I understand the changelog correctly: https://chevrotain.io/docs/changes/BREAKING_CHANGES.html#_7-0-0

Reply:
okay